fal/controllers/anime_stats.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+
+import configparser
+from typing import TYPE_CHECKING, Dict, Union, List, cast
+
+import jikanpy
+
+from fal.clients.mfalncfm_main import session_scope
+from fal.models import AnimeWeeklyStat, Season, Anime
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Session
+
+config = configparser.ConfigParser()
+config.read("config.ini")
+
+
+def get_forum_posts(anime: Anime) -> int:
+    '''Not yet implemented'''
```

Comment:
Would it be better if we raised a NotImplementedError here? I think it would be safer.

Reply:
I could do that and remove the call to `get_forum_posts` below and just replace that with 0. Would that work better?

Reply:
I do plan on implementing this real soon though

Reply:
Oh okay, then it's fine.